$OpenBSD: patch-configure,v 1.28 2013/06/21 22:42:13 sthen Exp $

Our crypt(3) is in libc.

Use uuid from sysutils/e2fsprogs instead of devel/uuid
(ECKrbAuth.cpp:61 et/com_err.h).

--- configure.orig	Thu Feb 28 16:13:46 2013
+++ configure	Tue Jun 18 13:09:00 2013
@@ -18780,74 +18780,12 @@ rm -f core conftest.err conftest.$ac_objext \
 
 LIBS=$LIBS_system
 
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for crypt" >&5
-$as_echo_n "checking for crypt... " >&6; }
-cat confdefs.h - <<_ACEOF >conftest.$ac_ext
-/* end confdefs.h.  */
- int main() {
-  char crypt();  crypt();
-}
-_ACEOF
-if ac_fn_cxx_try_link "$LINENO"; then :
-   { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
-$as_echo "yes" >&6; }
-else
-   { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-	  { $as_echo "$as_me:${as_lineno-$LINENO}: checking for crypt in -lcrypt" >&5
-$as_echo_n "checking for crypt in -lcrypt... " >&6; }
-if test "${ac_cv_lib_crypt_crypt+set}" = set; then :
-  $as_echo_n "(cached) " >&6
-else
-  ac_check_lib_save_LIBS=$LIBS
-LIBS="-lcrypt  $LIBS"
-cat confdefs.h - <<_ACEOF >conftest.$ac_ext
-/* end confdefs.h.  */
-
-/* Override any GCC internal prototype to avoid an error.
-   Use char because int might match the return type of a GCC
-   builtin and then its argument prototype would still apply.  */
-#ifdef __cplusplus
-extern "C"
-#endif
-char crypt ();
-int
-main ()
-{
-return crypt ();
-  ;
-  return 0;
-}
-_ACEOF
-if ac_fn_cxx_try_link "$LINENO"; then :
-  ac_cv_lib_crypt_crypt=yes
-else
-  ac_cv_lib_crypt_crypt=no
-fi
-rm -f core conftest.err conftest.$ac_objext \
-    conftest$ac_exeext conftest.$ac_ext
-LIBS=$ac_check_lib_save_LIBS
-fi
-{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_cv_lib_crypt_crypt" >&5
-$as_echo "$ac_cv_lib_crypt_crypt" >&6; }
-if test "x$ac_cv_lib_crypt_crypt" = x""yes; then :
   cat >>confdefs.h <<_ACEOF
 #define HAVE_LIBCRYPT 1
 _ACEOF
 
-  LIBS="-lcrypt $LIBS"
 
-else
-  as_fn_error $? "required library libcrypt missing or unusable" "$LINENO" 5
-fi
 
-fi
-rm -f core conftest.err conftest.$ac_objext \
-    conftest$ac_exeext conftest.$ac_ext
-CRYPT_LIBS=$LIBS
-LIBS=$LIBS_system
-
-
 # Only specific links with uuid and dl, clients only with crypto and ssl for ssl connections
 SSL_LIBS=
 # TODO: version of ssl check??
@@ -18858,7 +18796,7 @@ if test "${ac_cv_lib_ssl_SSL_accept+set}" = set; then 
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lssl  $LIBS"
+LIBS="-lssl -lcrypto $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
